spread.yaml

```diff
@@ -21,11 +21,11 @@ prepare: |
     apt build-dep -y ./
     test -d /home/test || adduser --quiet --disabled-password --gecos '' test
     chown test.test -R ..
-    sudo -i -u test /bin/sh -c "cd $PWD && dpkg-buildpackage -b -Zgzip"
+    sudo -i -u test /bin/sh -c "cd $PWD && DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -b -Zgzip"
```

Comment:
Because that's prepare. We shouldn't have to run these tests every time we want to test something else. Instead, we should have a specific test covering whatever is being tested there.